### PR TITLE
UI and UI functionality changes for AB#12304.

### PR DESCRIPTION
**Pull request by BrianMaki**
# Fixes or Implements [AB#12304](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/12304).

## Description

1. Sidebar not shown when logged in
2. Confirmed integration with backend - manual with no ID passed in, manual with ID passed in, authenticated with no ID passed in and authenticated with ID passed in.
3. Changed values as per requirements in time taken drop down box
4. Added PHN validation
5. Added Test Kit Code and validation for Test Kit Code field
6. Fixed validation on existing form input fields
7. Fixed Full Name not shown in logged in scenario

Screen Shot 2022-02-07 at 4 07 12 PM

Screen Shot 2022-02-07 at 4 07 56 PM

Screen Shot 2022-02-07 at 4 08 48 PM

Screen Shot 2022-02-07 at 4 09 01 PM

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

-   [ ] Unit Tests Updated
-   [ ] Functional Tests Updated
-   [x] Not Required

### UI Changes

YES

### Browsers Tested

-   [x] Chrome
-   [ ] Safari
-   [ ] Edge
-   [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant. Include any specialized deployment steps here.

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

-   Fulfills Work Item Requirements

    -   The changes meet/implement story's acceptance criteria. Fixes identified problems if bug.
    -   Changes not directly related to the task requirements need to be documented on the PR.

-   Compilation / Tests

    -   The changes do not break compilation and/or unit or functional tests; unless other item addresses them specifically.

-   Logic Problems / Functional Behaviour

    -   The changes work as intended and do not introduce problems.

-   Performance

    -   The changes do not introduce obvious performance issues.

-   Documented Standards

    -   The changes adhere to the team's documented [Coding Standards](https://github.com/bcgov/healthgateway/wiki/standards).

-   Readability / Maintainability
    -   The changes can be easily understood/read and allow for future enhancements without major refactoring. Readability is preferred over "clever code".
    -   Reasoning for changes needs to be clearly articulated. Disagreements should be arbitrated by a third developer.

### CODE REVIEW

File: Apps/WebClient/src/ClientApp/src/views/pcrTest.vue [hunk 1]
```diff
@@ -230,8 +206,8 @@ export default class PCRTest extends Vue {
         return smsMask;
     }
 
-    private get fullName(): string {
-        if (this.oidcIsAuthenticated) {
+    private getfullName(): string {
```
Comment (MikeLyttle) [inline Apps/WebClient/src/ClientApp/src/views/pcrTest.vue:209]: Can we rename this to `getFullName()` with camel case capitalization?

File: Apps/WebClient/src/ClientApp/src/models/registerTestKitRequest.ts [hunk 2]
```diff
@@ -6,5 +6,11 @@ export default interface RegisterTestKitRequest {
     testTakenMinutesAgo?: number;
 
     // ID of the test kit
-    testKitId?: string;
+    testKitCid?: string;
```
Comment (pagustinqt20) [inline Apps/WebClient/src/ClientApp/src/models/registerTestKitRequest.ts:9]: Why the field name was change to testKidCid?

File: Apps/WebClient/src/ClientApp/src/views/pcrTest.vue [hunk 3]
```diff
@@ -573,44 +582,45 @@ export default class PCRTest extends Vue {
...
+                                    @blur.native="
+                                        $v.pcrTest.testKitCode.$touch()
+                                    "
                                 />
                                 <b-form-invalid-feedback
                                     v-if="
-                                        $v.pcrTest.testKitId.$dirty &&
-                                        !$v.pcrTest.testKitId.required
+                                        $v.pcrTest.testKitCode.$dirty &&
+                                        !$v.pcrTest.testKitCode.formatted
```
Comment (MikeLyttle) [inline Apps/WebClient/src/ClientApp/src/views/pcrTest.vue:612]: It looks like we display an error when the test kit fails the formatted validator but not when it fails the required validator.

File: Apps/WebClient/src/ClientApp/src/models/registerTestKitRequest.ts (hunk 2 above)
Comment (BrianMaki) [reply Apps/WebClient/src/ClientApp/src/models/registerTestKitRequest.ts:9]: That is what it is at PHSA.

**Review by pagustinqt20 — APPROVED** [review]

**Review by MikeLyttle — APPROVED** [review]

**Pull request merged**